### PR TITLE
feat: added files for badge component

### PR DESCRIPTION
**Pull request by Ranjeetk489**

### CODE REVIEW

**Review by sanu29 — APPROVED** [review]
Approved. Looks clean and structured

File: components/Badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+  vertical-align: middle;
+  position: relative;
+}
+
+.badge-envelope .mail:after {
+  content: "77";
+  position: absolute;
+  background: var(--error-color);
+  height: 1.915rem;
+  top: -0.9rem;
```
Comment (abhishektanwar) [inline components/Badge/badge.css]: px could be used if value with relative unit result is < 10px

File: components/Badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+  text-align: center;
+  line-height: 1.1rem;
+  font-size: 10px;
+  border-radius: 50%;
+  /* color: white; */
+  border: 1px solid var(--tertiary-color);
+}
+
+.badge-envelope {
+  width: 6.25rem;
```
Comment (abhishektanwar) [inline components/Badge/badge.css]: Please try to remain consistent with spacing between your css classes and please remove comments.

**Review by abhishektanwar — APPROVED** [review]
Good to merge.

**Pull request merged**